modin/pandas/test/test_groupby.py

```diff
@@ -1968,6 +1968,21 @@ def compare(obj1, obj2):
     compare(reference_by, result_by)
 
 
+@pytest.mark.parametrize("width", [2**5, 2**10])
+@pytest.mark.parametrize("height", [2**5, 2**10])
+@pytest.mark.parametrize("concat_length", [5, 10])
+def test_concat_groupby(width, height, concat_length):
+    # from https://github.com/modin-project/modin/issues/4464
+    data = np.random.randint(0, 100, size=(width, height))
```

Comment:
Let's use `test_data["int_data"]`. We shouldn't generate random data without a seed because we want tests to be deterministic. The `test_data` is deterministic and should be large enough.